src/test/scala/com/bazaarvoice/sswf/CancelTest.scala

```diff
@@ -23,7 +23,7 @@ class CancelTestWorkflowDef(rememberer: Rememberer) extends WorkflowDefinition[S
 }
 
 class Rememberer {
-  var toRemember: String = null
+  var toRemember: String = _
```

Comment:
you can actually use '???' if you want it to blow up immediately instead of potentially injecting a null value into your logic flow.